src/FormBuilder/Routes/RegisterFormBuilderPageRoute.php

```diff
@@ -117,6 +105,12 @@ public function renderPage()
             true
         );
 
+        wp_add_inline_script(
+            '@givewp/form-builder/script',
+            'window.giveStorageData = ' . wp_json_encode($formBuilderViewModel->storageData($donationFormId)) . ';',
```

Comment:
This change is breaking the preview URL - something is different about how the data is localized.

Reply:
`wp_localize_script` passes each value through `html_entity_decode( (string) $value, ENT_QUOTES, 'UTF-8' )` before attaching the data to the window.

Reply:
This seems to be a conflict with calling `esc_url` up the line.

Reply:
Resolved in [3f7cff2](https://github.com/impress-org/givewp/pull/6911/commits/3f7cff25a8bc156a87e0f28923acdb431ea7bf4e)